lib/kino_db/connection_cell.ex

```diff
@@ -7,30 +7,37 @@ defmodule KinoDB.ConnectionCell do
   use Kino.JS.Live
   use Kino.SmartCell, name: "Database connection"
 
-  @default_port_by_type %{"postgres" => 5432, "mysql" => 3306}
+  @default %{
+    "database" => %{"athena" => "default"},
+    "hostname" => %{"postgres" => "localhost", "mysql" => "localhost"},
+    "port" => %{"postgres" => 5432, "mysql" => 3306},
+    "credentials" => %{"bigquery" => %{}}
+  }
+
+  @attr_keys ~w|
+    hostname port database username password database_path
+    project_id default_dataset_id credentials
+    access_key_id secret_access_key region output_location
+  |
 
   @impl true
   def init(attrs, ctx) do
     type = attrs["type"] || default_db_type()
-    default_port = @default_port_by_type[type]
-
-    fields = %{
-      "variable" => Kino.SmartCell.prefixed_var_name("conn", attrs["variable"]),
-      "type" => type,
-      "hostname" => attrs["hostname"] || "localhost",
-      "database_path" => attrs["database_path"] || "",
-      "port" => attrs["port"] || default_port,
-      "username" => attrs["username"] || "",
-      "password" => attrs["password"] || "",
-      "database" => attrs["database"] || "",
-      "project_id" => attrs["project_id"] || "",
-      "default_dataset_id" => attrs["default_dataset_id"] || "",
-      "credentials" => attrs["credentials"] || %{},
```

Comment:
With the new approach I think we will end up with `"credentials => ""` unless we start with big query?

To be honest I prefer the current approach because it's simpler and we wouldn't be saving much on the actual code :)